hrm-service/src/contact/sql/contact-get-columns-for-transcripts-sql.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Given a date range (dateFrom & dateTo),
+ * returns the required columns to transcript a contact chat ("accountSid", "serviceSid", "channelSid")
+ * for all of the contacts created in that range.
+ */
+export const selectColumnsForTranscript = () => `
+  SELECT "accountSid", "serviceSid", "channelSid" 
+  FROM "Contacts" 
+  WHERE ("serviceSid" != '') IS TRUE
+    AND ("channelSid" != '') IS TRUE
+    AND (
+      $<dateFrom> IS NULL OR "createdAt" >= $<dateFrom>
+      -- $<dateFrom> IS NULL OR "timeOfContact" >= $<dateFrom>
+    )
+    AND (
+      $<dateTo> IS NULL OR "createdAt" <= $<dateTo>
+      -- $<dateTo> IS NULL OR "timeOfContact" <= $<dateTo>
+    );
+`;
```

Comment:
About this query, two comments:
- `(stringStatement != '') IS TRUE` is a way to check emptiness for strings (not NULL nor empty string).
- I think we want to use `createdAt` here but let me know if you think `timeOfContact` is better for this. `timeOfContact` can only differ for offline contacts tho, and here we only care about chat contacts.